tests/cli/test_train_model.py

```diff
@@ -153,7 +153,10 @@ def test_train_unknonw_arch_options(monkeypatch, tmp_path):
     """
     options = OmegaConf.create(options_str)
 
-    match = r"Additional properties are not allowed \('num_epoch' was unexpected\)"
+    match = (
+        r"Additional properties are not allowed \('num_epoch' was unexpected\). "
```

Comment:
I wouldn't use the "additional properties" expression in the errors. The user has no clue what "additional properties" are. I would rather go for "unrecognized option" or something along those lines.